QA.md

```
@@ -120,21 +121,52 @@ last_verified_at: 2026-08-21
+- [x] **iOS で左エッジスワイプでも戻れる**: push 遷移した画面 (ホーム以外のほぼ全画面) で、画面左端からのスワイプで前の画面に戻れる
+  - 自動化: NikkiUITests/NavigationSwipeBackUITests.swift の `testSwipeBackReturnsToPreviousScreen`（通常フロー(RootPage)配下でホーム → 設定へ push し、左エッジのドラッグで戻ることを実イベントで検証する。CI では実行されない）
+  - 2026-08-25 ローカル iOS Simulator で pass。simtunnel リモート iOS Simulator (iOS 26) の手動操作でも、エディタからのスワイプでホームへ戻れた (https://pub-7f3469dd3e2e445b9b8ec2d1381b5ea8.r2.dev/bannzai/nikki/20260825/d619de59-47dd-4fa1-94ff-52f8d327b20e.jpg)
+  - 補足: issue #92 時点で「simtunnel の WDA 経由の合成タッチでは検証不可」と記録していたが、システムのナビゲーションバーへ移行した後は WDA の `swipe` でも戻れることを確認した (エッジ判定に入る最左端から始め、確定までゆっくり動かす)
+- [x] **iOS で戻るボタンのタップでも戻れる**: push 遷移した画面のシステムの戻るボタンをタップすると前の画面に戻る
+  - 自動化: NikkiUITests/NavigationSwipeBackUITests.swift の `testBackButtonReturnsToPreviousScreen`（CI では実行されない）
+  - 2026-08-25 ローカル iOS Simulator で pass。自動ロックの無操作検出を旧実装 (`simultaneousGesture` の `DragGesture`) に一時的に戻すとこのテストだけが失敗することを確認済みで、戻るボタンのタップが奪われる回帰を実際に検出できる (スワイプ側は旧実装でも pass するため、この項目が回帰の検出を受け持つ)
+- [x] **macOS で戻るボタンのタップでも戻れる**: ウィンドウツールバーの戻るボタンをタップすると前の画面に戻る
+  - 自動化: manual（macOS の UI テストは provisioning profile が無くローカルで実行できないため、実操作で確認する）
+  - 2026-08-25 ad-hoc 署名の Debug ビルドで、ホームの歯車から設定へ push し (https://pub-7f3469dd3e2e445b9b8ec2d1381b5ea8.r2.dev/bannzai/nikki/20260825/daf2dc18-8bba-48eb-a4ac-c3cddf130af2.png)、戻るボタンでホームへ戻れた (https://pub-7f3469dd3e2e445b9b8ec2d1381b5ea8.r2.dev/bannzai/nikki/20260825/ecb4c063-4a18-4246-b17c-0292f002c79d.png)
 
 #### 動作確認
```

markdownlint-cli2 (0.23.2)

[warning] Heading levels should only increment by one level at a time
Expected: h3; Actual: h4

(MD001, heading-increment)